_posts/2017/2017-10-30-reunification_beta.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Reunification beta 1"
+author: dbarrosop
+description: "New napalm with all the unified drivers is in beta. Come and check it out."
+tags: ["news"]
+---
+
+As we announced in a [previous post](/reunification/) we have unified all of our core drivers under the same [napalm repo](https://github.com/napalm-automation/napalm). Right now, we have released a beta version which we hope you can help us testing. To do so just run the following ``pip`` command:
+
+```
+pip install --pre napalm  # --pre is only necessary while this is in beta
+```
+
+A continuation some useful notes.
```

Comment:
I'm not sure I understand that.

Reply:
yeah, that makes sense if you are seeing the list of blog entries as it sits right on top of "continue reading". I will remove.